karols-frontend/src/containers/ResumeContainer.js

```diff
@@ -21,6 +21,23 @@ const mapStateToProps = state => ({
 })
 
```

Comment:
tthe method showAlert is not necessary, you don't need the visibiliy in state.
What you need is to connect (mapstatetoprops) a showAlert prop with the reservation.success in the store
and you use this prop to show or not the alert